src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -347,9 +422,9 @@ public Model clone() {
     public void setContext(ModelContext context) {
         switch (context) {
         case CONTEXT_LIST:
-            displayEntryBook(this.listEntryBook);
+            setDisplayEntryList(this.listEntryBook);
             break;
-        case CONTEXT_ARCHIVE:
+        case CONTEXT_ARCHIVES:
             // something else
```

Comment:
Does the context switch to `ARCHIVES` entail the archives `EntryBook` being displayed?
If yes shouldn't we do it here?

Reply:
Good catch. I missed out this function here in doing this PR. We should probably do it here, to be honest, to make it more defensive such that on a command-triggered context-switch, we are also guaranteed the display is updated.